__tests__/pages/provinces/index.test.tsx

```diff
@@ -58,26 +57,6 @@ describe("ProvincesPage", () => {
     ).toBeVisible();
   });
 
-  it.each`
-    kebutuhan           | count
-    ${"Ambulans"}       | ${provinceListItem.ambulansCount}
-    ${"Rumah%20sakit"}  | ${provinceListItem.rsCount}
-    ${"Oksigen"}        | ${provinceListItem.oksigenCount}
-    ${"Donor%20plasma"} | ${provinceListItem.donorPlasmaCount}
-  `(
-    "renders the entry count of kebutuhan $kebutuhan in province list correctly",
-    async ({ kebutuhan, count }) => {
-      await router.push(`/provinces?kebutuhan=${kebutuhan}`);
-      render(<ProvincesPage provincesList={provinceList} />);
-
-      const provinceLink = screen.getByRole("link", {
-        name: /informasi faskes & alkes untuk covid-19 di provinsi/i,
-      });
-
-      expect(within(provinceLink).getByText(`${count} Entri`)).toBeVisible();
-    },
-  );
-
```

Comment:
Why did you delete this test?

Reply:
I didn't. I never change other files beside telekonseling files actually. Is this happens when I run ```yarn run test:coverage -u``` locally?